packages/dialtone-vue2/common/emoji/index.js

```diff
@@ -1,5 +1,5 @@
 import { emojiPattern } from 'regex-combined-emojis';
-import emojiJsonLocal from 'emoji-toolkit/emoji_strategy.json' with { type: 'json' };
+import { emojisIndexed } from '@dialpad/dialtone-emojis';
```

Comment:
Nice, this is much better to not have to reference emoji-toolkit anymore. However, there is some process and/or script that nacho has to transfer emoji data from emoji-toolkit into these categories. We will have to perform this whenever we want to update to a newer version of emojis. I'll make a note to get this from him when he's back

Reply:
Ah right, I was wondering why we were not using  `@dialpad/dialtone-emojis` on Dialtone-vue 2 version.

So should I remove this change? ESLint was complaining so just figure out updating it to dialtone-emojis fixed it.

Reply:
Nah should be okay to make this change